### DISCUSSION TITLE
이동속도 버프를 주는 비콘을 만들래요! [치자피즈]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#169] [3. Creator Samples]
간단하게 구연할 수 있습니다!
단 한줄로 쉽게 이동속도를 추가하거나 추가한 수치를 원상복구시킬 수 있어요!

이 문장 하나면 완성이죠!
ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character.additionalRunSpeed = 5;

1. 이동속도 증가 비콘을 제작합니다.
2. 이동속도 증가 해제 비콘을 제작합니다.
3. 완성!

<img width="892" alt="스크린샷 2022-01-05 오후 8 18 05" src="https://user-images.githubusercontent.com/90679926/148209388-2db89154-88bc-4755-aed5-6e9c08683673.png">

<img width="892" alt="스크린샷 2022-01-05 오후 8 18 07" src="https://user-images.githubusercontent.com/90679926/148209394-1931c6b9-5502-4452-9cfa-69bde1a5fb6b.png">

<img width="1051" alt="스크린샷 2022-01-05 오후 8 18 27" src="https://user-images.githubusercontent.com/90679926/148209398-4bd1dc63-a458-4f4e-8246-324704a9be7a.png">

걷기, 달리기, 뛰기를 제외한 버프도 만들 수 있을까요? 여러분들은 어떤 아이디어가 있나요?

틀린 정보가 있다면 댓글달아주세요 :D

**Comment by castpuz** [comment]
안녕하세요? 재미있는 기능이네요, 근데 additionalRunSpeed 같은 함수는 어디서 찾을 수 있나요? 가이드에는 안보이는거같은데..

**Reply by x4cf** [reply]
탐험한다는 생각으로 설레는 마음을 가지고 페키지 안의 클래스를 탐구하면 알 수 있어요 :D

**Reply by UnpaidTax** [reply]
아 멘트 지렸어요 리스펙 근데 아직 잘볼줄몰라서요
